experimental/auth-logic/top_level.go

```diff
@@ -87,7 +87,7 @@ func verifyRelease(appName, endorsementFilePath, provenanceFilePath string) (str
 	}
 
 	topLevelQuery := "verification_success = query " + verifierPrincipal.String() +
-		" says \"" + appName + "::Binary\" canActAs " + appName + "?"
+		" says \"" + appName + "::Binary\" canActAs \"" + appName + "\"?"
```

Comment:
I think `fmt.Sprintf` with `%q` could solve the quotation problem.

Reply:
fixed